apps/server/src/modules/learnroom/controller/rooms.controller.ts

```diff
@@ -69,6 +69,11 @@ export class RoomsController {
 		@Param() urlParams: RoomUrlParams
 	): Promise<CopyApiResponse> {
 		const copyStatus = await this.courseCopyUc.copyCourse(currentUser.userId, urlParams.roomId);
+		const newElements = copyStatus.elements?.filter((element) => {
+			const filterCopyBoard = element.type !== CopyElementType.LESSON_CONTENT_NEXBOARD;
+			return filterCopyBoard;
+		});
+		copyStatus.elements = newElements;
```

Comment:
This is not the right place to do it - you're just filtering the output for the caller so the content of the course will be copied anyway. You have to dig deeper to prevent making a physical copy of the Nexboards, not just not showing the copy that has been made anyway (but shouldn't).